### PR TITLE
ESC telemetry page.

### PR DESCRIPTION
**Pull request by pulquero**
Added ESC (read-only) telemetry page.

### CODE REVIEW

File: src/SCRIPTS/BF/X7/x7pre.lua [hunk 1]
```diff
@@ -8,7 +8,8 @@ PageFiles =
     "filters1.lua",
     "filters2.lua",
     "pwm.lua",
-    "vtx.lua"
+    "vtx.lua",
+    "esc.lua",
```
Comment (mikeller) [inline src/SCRIPTS/BF/X7/x7pre.lua]: Shouldn't this go into `X9/` and `HORUS/` as well?

**Comment by codecae** [comment]
Given that this is a read-only extension, would it be better served up as a separate telemetry script?

**Comment by pulquero** [comment]
The limit on the total number of scripts made me think I should be conservative in this respect.

File: src/SCRIPTS/BF/X7/x7pre.lua (hunk 1 above)
Comment (pulquero) [reply src/SCRIPTS/BF/X7/x7pre.lua]: Sure, once you are happy with my approach I'll roll it out to the other radios too.

**Comment by mikeller** [comment]
@codecae: How easy is it to do this as a separate telemetry script, but reuse the existing MSP functionality? Copying / reimplementing these parts would be very inefficient.

**Comment by codecae** [comment]
The common map components can be reused.  They were separated from the UI for this reason.  All that would be required is the creation of a new event loop

**Comment by codecae** [comment]
Using the existing UI may end up saturating the signal due to constant polling.  It could be better controlled with a purpose built event loop.

**Comment by pulquero** [comment]
Closing in favour of https://github.com/betaflight/betaflight/pull/5581.

**Pull request closed without merging**